diffs/packetbeatdiff.diff

```diff
@@ -0,0 +1,2602 @@
+            </dl>
+           </div>
+           <h3>
+-           <a id="_fingerprints">
++           <a id="_fingerprints_2">
+            </a>
+            fingerprints
+            <a class="edit_me" href="https://github.com/elastic/beats/edit/master/packetbeat/docs/fields.asciidoc" rel="nofollow" title="Edit this page on GitHub">
+html/en/beats/packetbeat/master/exported-fields-trans_event.html
+html/en/beats/packetbeat/master/exported-fields-trans_measurements.html
+html/en/beats/packetbeat/master/exported-fields.html
+html/en/beats/packetbeat/master/extract-array.html
+html/en/beats/packetbeat/master/faq.html
+html/en/beats/packetbeat/master/feature-roles.html
+html/en/beats/packetbeat/master/file-output.html
+--- /dev/fd/63
++++ html/en/beats/packetbeat/master/file-output.html
+@@ -214,7 +214,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_enabled_literal_8">
++                <a id="_enabled_8">
+                 </a>
+                 <code class="literal">
+                  enabled
+@@ -259,7 +259,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_filename_literal">
++                <a id="_filename">
+                 </a>
+                 <code class="literal">
+                  filename
+@@ -280,7 +280,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_rotate_every_kb_literal">
++                <a id="_rotate_every_kb">
+                 </a>
+                 <code class="literal">
+                  rotate_every_kb
+@@ -301,7 +301,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_number_of_files_literal">
++                <a id="_number_of_files">
+                 </a>
+                 <code class="literal">
+                  number_of_files
+@@ -328,7 +328,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_permissions_literal">
++                <a id="_permissions">
+                 </a>
+                 <code class="literal">
+                  permissions
+@@ -349,7 +349,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_codec_literal_3">
++                <a id="_codec_3">
+                 </a>
+                 <code class="literal">
+                  codec
+html/en/beats/packetbeat/master/filtering-and-enhancing-data.html
+html/en/beats/packetbeat/master/getting-help.html
+html/en/beats/packetbeat/master/http-endpoint.html
+html/en/beats/packetbeat/master/ilm.html
+--- /dev/fd/63
++++ html/en/beats/packetbeat/master/ilm.html
+@@ -283,7 +283,7 @@
+           <p>
+            The index lifecycle write alias name. The default is
+            <code class="literal">
+-            packetbeat-{agent.version}

```

Comment:
fixed in https://github.com/elastic/beats/pull/13722/commits/def5e8ef969937918badbd5ddf624addb3f3536c